site/_layouts/post.latte

```diff
@@ -1,4 +1,4 @@
-{extends "default"}
+{extends "_layouts/default.latte"}
```

Comment:
Full relative paths are used now, to prevent "default" vs "default" file conflicts